inc/api.php

```diff
@@ -81,12 +83,12 @@ function get_body() : array {
 
 	if ( ! $body ) {
 		$response = get_data();
-		$code = get_code();
-		$expire = DashboardChangelog\get_cache_expiration();
+		$code     = get_code();
```

Comment:
No, but nothing prevents them either, in the rulest. I probably have aligned the "=" I guess.

Reply:
I don't have a strong enough opinion about spacing. I'm not going to block the PR for that. I generally don't align `=` because if you change things later, you end up spending time moving the `=`s around instead of writing actual code.